docker_and_oci.rst

```diff
@@ -419,6 +421,16 @@ Ubuntu image for a 64-bit ARM system:
 
    $ {command} pull --arch arm64 docker://ubuntu
 
+``--arch-variant`` option
+=========================
+
+The ``--arch-variant`` option can be used for CPU variants. Currently only the
+ARM CPU variants are supported.
```

Comment:
Is it just a small number of variants as well?  If so I think it would help to list them here.

Shouldn't it also mention that the variant option is only supported for the `build` command?

Reply:
It is very small, for arm32 there is 5 and 6 7 - and armv5 isn't used in many places...

But --arch-variant should also work for `run`, just that it was undocumented before?

Reply:
Sorry, I actually meant to say `pull` there rather than `build` - my bad.